extension/manifest.json

```diff
@@ -2,27 +2,40 @@
     "manifest_version": 2,
     "name": "BiReader",
     "version": "1.1",
-
     "description": "Based off of the principles of Bionic Reader to improve speed and depth of reading web text!",
-
     "icons": {
         "48": "icons/48.png",
         "96": "icons/96.png"
     },
-
     "browser_action": {
         "default_icon": "icons/32.png",
         "default_title": "Turn BiReader On"
     },
-
     "background": {
-        "scripts": ["browser-polyfill.js", "background.js"]
+        "scripts": [
+            "background.js"
+        ]
```

Comment:
These lines don't need to be reformatted

Reply:
This file is actually autogenerated from `scripts/generateManifest.js` (which contains a function to return a manifest object based on the browser, so it doesn't have to do patching stuff) - right now the content of the file is produced with `JSON.stringify(/* manifest object */, null, 4)`, I could make it not format the file and put it all in one line if that would be better?